main/create_simulated_body_model.py

```
@@ -0,0 +1,1085 @@
+    QStatusBar,
+    QCheckBox,
+)
+
+from nav.sim.render import render_combined_model
+
+
+class ImageLabel(QLabel):
+    def __init__(
+        self,
+        parent: Optional[QWidget],
+        on_press: Any,
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `on_press`

(ANN401)